core_v2/sources/domains.move

```diff
@@ -53,6 +55,16 @@ module aptos_names_v2::domains {
     const EVALID_SIGNATURE_REQUIRED: u64 = 16;
     /// The domain is too short.
     const EDOMAIN_TOO_SHORT: u64 = 17;
+    /// The name is not expired in 6 months, thus not eligible for renewal
+    const EDOMAIN_NOT_AVAILABLE_TO_RENEW: u64 = 18;
+    /// The subdomain is not eligible for renewal
+    const ESUBDOMAIN_IS_AUTO_RENEW: u64 = 19;
```

Comment:
can we rename this to `ESUBDOMAIN_IS_NOT_AVAILABLE_AUTO_RENEW`

Reply:
this error is about that this subdomain is auto renew so we can't manually set a expiration date on it. I also think the name is a bit bad, but can't think of a better one. any suggestion?

Reply:
i seem maybe `ESUBDOMAIN_IS_AUTO_RENEW` is fine